### PR TITLE
Localize the modified column

### PR DESCRIPTION
**Pull request by janus2**

### CODE REVIEW

**Comment by humdingerb** [comment]
Nice!

But what about our coding style...? :)

**Comment by janus2** [comment]
Do we have a coding style? This is a cut and paste from Haiku with minor changes...

File: Main/DateColumn.cpp [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ * Distributed under the terms of the MIT license.
+ *
+ * Authors:
+ *		Janus
+ */
+
+/*
+	Based on Tracker Modified Column
+*/
+
```
Comment (humdingerb) [inline Main/DateColumn.cpp]: Two newlines here.

File: Main/DateColumn.cpp [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+#include <DateFormat.h>
+#include <DateTimeFormat.h>
+#include "DateColumn.h"
```
Comment (humdingerb) [inline Main/DateColumn.cpp]: One newline above.

File: Main/DateColumn.cpp [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+
+#define kTEXT_MARGIN	8
+
+
+DateColumn::DateColumn	(const char* title,
+								 float width,
+								 float minWidth,
+								 float maxWidth,
+								 alignment align ):
```
Comment (humdingerb) [inline Main/DateColumn.cpp]: Not sure about the large indention, and the 1 parameter/line, but I don't mind much...
What is amiss, is the space in front of the bracket: "alignment align ):"

File: Main/DateColumn.cpp [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+		BDateColumn(title,width,minWidth,maxWidth,align){}
```
Comment (humdingerb) [inline Main/DateColumn.cpp]: on newlines:
{
}

File: Main/DateColumn.cpp [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+								
+void
+DateColumn::DrawField(BField* _field, BRect rect, BView* parent)
+{
+	time_t value=(int)((BDateField*)_field)->UnixTime();
+	if(value<=0) return;
+
```
Comment (humdingerb) [inline Main/DateColumn.cpp]: spaces arounf "=" and "<="
"return" in new line

File: Main/DateColumn.h [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+	public:
+					DateColumn	(const char* title,
+								 float width,
+								 float minWidth,
+								 float maxWidth,
+								 alignment align = B_ALIGN_LEFT);
+		  void		DrawField	(BField* field,
+								 BRect rect,
+								 BView* parent);
+};
```
Comment (humdingerb) [inline Main/DateColumn.h]: The indentation in the rest or PecoRename's .h are different. It's probably not "real" Haiku code style, but for consistency's sake...

**Pull request merged**